src/test/DynamicFeesTestHook.sol

```diff
@@ -30,10 +31,10 @@ contract DynamicFeesTestHook is BaseTestHooks {
     function beforeSwap(address, PoolKey calldata key, IPoolManager.SwapParams calldata, bytes calldata)
         external
         override
-        returns (bytes4, int128)
+        returns (bytes4, BeforeSwapDelta)
     {
         manager.updateDynamicLPFee(key, fee);
-        return (IHooks.beforeSwap.selector, 0);
+        return (IHooks.beforeSwap.selector, BeforeSwapDelta.wrap(0));
```

Comment:
use constant

Reply:
& i think theres a few more places that could just use BeforeSwapDelta.ZERO_DELTA